docs/setup/developer.md

```diff
@@ -1,6 +1,34 @@
 # Developer setup
 
-## Installation
+## Setting up Python virtual environment
+
+### Using conda
+```
+conda create -n "pixlVE" python=3.11  pip -c conda-forge
```

Comment:
out of interest, what's the `VE` here?

Reply:
sorted our here e30ee053808506446244759de74d6fe4dee229eb